docs/modules/mobile/cloud-deploy.md

```diff
@@ -52,9 +54,11 @@ See [this page]({{urlRoot}}/modules/mobile/run-client) for more information on t
 1. Select **SpatialOS** > **Mobile Launcher** to open the Mobile Launcher window.
 1. Fill in your **Development Team ID**. (There are instructions on where to find it on this [Stackoverflow topic](https://stackoverflow.com/a/47732584).)
 1. Select **Build XCode project** and wait for the build to complete.
-1. Select your device or simulator from the relevant dropdown.
-   * If your device or simulator does not show up, you can select the button to the right of the dropdown to refresh the list.
-1. Select the **Launch iOS app on Simulator** or **Launch iOS app on Device**
+1. Select your Simulator or device from the relevant dropdown.
```

Comment:
Why did you reverse the order?

Reply:
for consistency with the section headings + the order in which we display them in the mobile launcher itself